data-raw/healthy-places/living-conditions/rough-sleeping.R

```diff
@@ -0,0 +1,95 @@
+# Installing relevant packages
+library(tidyverse)
+library(readxl)
+library(httr)
+library(dplyr)
+library(geographr)
+
+# Extracting relevant data sets from statswalesr
+# library(statswalesr)
+# statswales_search(c("rough sleeping"))
+# statswales_search(c("homelessness"))
+
+# 
+# # Scrape data from stats Wales
+# # Source:https://statswales.gov.wales/v/P2SD
+# df <- statswales_get_dataset("HOUS0454")
+
+# Upload csv files for rough sleepers and population data
+#read.csv()
+#df <- rough_sleepers_data
+#head(population_df)
+#df <- rough_sleepers_data
+
+# Need to filter to only include Welsh data (mid-2023)
+# population_df <- population_df |>
+#   filter(location == "Wales")
+# str(population_df)
+# head(population_df)
+
+population_df_wales <- population_df |>
```

Comment:
Please see [here](http://adv-r.had.co.nz/Style.html) for how to name objects. 

Typically when we read a raw dataset (i.e. from the source) we add raw to the end of the object name e.g.` population_raw `

When we do transformations on datasets we then assign it a new object name e.g. `population_2023 <- population_raw |>` .  

We try not to rewrite over the same object multiple times when they are in different code blocks because it increases risk of error when you are running the script , you lose the original version of the data and it reduces readability of the code.  E.g. you rewrite the object `rough_sleepers_data ` multiple times. Note this isn't always achievable but is the general rule we stick to. You also don't need "df" or "wales" in your object name - unless it is meaningful.

Please review all the object names in the script.